.travis.yml

```diff
@@ -4,9 +4,25 @@ php:
   - 5.4
   - 5.5
 
+env:
+  - SYMFONY_VERSION="2.3.*"
+  - SYMFONY_VERSION="2.6.*"
+  - SYMFONY_VERSION="2.7.*"
+  - SYMFONY_VERSION="2.8.x-dev"
+  - SYMFONY_VERSION="3.0.x-dev"
+
+matrix:
+  allow_failures:
+    - env: SYMFONY_VERSION="2.8.x-dev"
+    - env: SYMFONY_VERSION="3.0.x-dev"
+
+
 before_script:
   - curl --silent http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction --prefer-source
+  - >
+      php composer.phar require --dev \
+        symfony/symfony:${SYMFONY_VERSION}
```

Comment:
always installing a ``symfony/symfony`` dependencies means that nothing ensure that the dependencies of the package are right, as you don't run any build with the unmodified deps.